bundle/config/validate/all_resources_defined.go

```diff
@@ -0,0 +1,47 @@
+package validate
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/databricks/cli/bundle"
+	"github.com/databricks/cli/libs/diag"
+	"github.com/databricks/cli/libs/dyn"
+)
+
+func AllResourcesHaveValues() bundle.Mutator {
+	return &allResourcesHaveValues{}
+}
+
+type allResourcesHaveValues struct{}
+
+func (m *allResourcesHaveValues) Name() string {
+	return "validate:all_resources_have_values"
```

Comment:
consistency: should be `validate.AllResourcesHaveValues `

Reply:
done